libdirections-common/build.gradle

```diff
@@ -26,9 +26,6 @@ dependencies {
 
     implementation dependenciesList.supportAnnotation
 
```

Comment:
`libdirections-common` module can be now removed as it's empty (should be removed also from https://github.com/mapbox/mapbox-navigation-android/pull/2240/files#diff-c7922c6bccdbff0138a25c167f5c86e5R23).

Reply:
yep, agree